### PR TITLE
[CE-1330] Escaping args

### PR DESCRIPTION
**Pull request by drazisil**
Sanitize gcov-root and ather args.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (hootener) [inline .vscode/launch.json]: I'm not sure this should be committed to source since it's IDE specific?

File: lib/codecov.js [hunk 2]
```diff
@@ -5,7 +5,6 @@ var urlgrey = require('urlgrey')
 var jsYaml = require('js-yaml')
 var walk = require('ignore-walk')
 var execSync = require('child_process').execSync
-var validator = require('validator')
```
Comment (hootener) [inline lib/codecov.js]: I'm unsure what benefits `validator` provides. But would it be worthwhile to keep it, and call it in your `sanitizeVar` function before the "&" removal step? Off the cuff, something like:

```
function sanitizeVar(arg) {
  arg = validator.escape(arg)
  return arg.replace(/&/g, '')
}
```

Comment (drazisil) [reply lib/codecov.js]: Validator was added with the last patch. It also escaped `/` which makes actual paths very unhappy.

File: .vscode/launch.json (hunk 1 above)
Comment (drazisil) [reply .vscode/launch.json]: No, this should not have been commited, good catch.

**Comment by drazisil** [comment]
@eddiemoore Can you please push a new release.

**Comment by drazisil** [comment]
This has been published under https://www.npmjs.com/package/codecov/v/3.6.5

**Pull request merged**